File: euler/1-10/2.c
#include <stdio.h>
int total;
int num1 = 1;
int num2 = 2;
int stor;
int main() {
	while (num2 < 4000000) {
		if (num2 % 2 == 0) {
			total += num2;
		}
		stor = num2;
		num2 += num1;
		num1 = stor;
	}
	printf("%d ", total);
}

File: euler/1-10/2.py
array = [1,1];
while array[-1] < 4000000:
    array.append(array[-1] + array[-2]);
del(array[-1]);
i=0;
total = 0;
while i < len(array):
    if array[i]%2==0:
        total += array[i];
    i+=1;
print(total);
    
